### ISSUE TITLE
return doesn't exit the block

### ISSUE THREAD

**Issue by matryer** [gobuffalo/plush#53]
If we use `return` somewhere (like in a conditional) execution does not exit the block as expected. Instead, it continues and future `return` statements seem to overwrite the previous returns.

For a real failing test example, see https://github.com/gobuffalo/plush/pull/52

**Comment by ruudk** [comment]
@matryer Tried to fix it in https://github.com/gobuffalo/plush/pull/59

**Comment by matryer** [comment]
Here's a test that fails, which shouldn't:

```go
func TestAdvancedCustomHelper(t *testing.T) {
	r := require.New(t)
	input := `
	
	<%
		let numberify = fn(arg) {
			if (arg == "one") {
				return 1
			}
			if (arg == "two") {
				return 2
			}
			if (arg == "three") {
				return 2
			}
			return "unsupported number"
		}
	%>
		<%= numberify("one") %>
	`
	s, err := Render(input, NewContext())
	r.NoError(err)
	r.Equal("1", strings.TrimSpace(s))
}
```

**Comment by Mido-sys** [comment]
@matryer, I believe I fixed the issue. I used the provided test cases along with the test cases from @ruudk pull's request.

**Comment by Mido-sys** [comment]
@paganotoni , Can you close this issue, please ? Fixed in [pull#127](https://github.com/gobuffalo/plush/pull/127)

**Comment by paganotoni** [comment]
For sure, thanks for the heads up @Mido-sys!
